packages/example-app-base/src/utilities/PlatformContext.tsx

```diff
@@ -6,12 +6,14 @@ export interface IWithPlatformProps<TPlatforms extends string = string> {
   platform?: TPlatforms;
 }
 
-export function withPlatform<TPlatforms extends string = string>(
-  Component: React.ComponentType
-): React.StatelessComponent<IWithPlatformProps<TPlatforms>> {
-  // tslint:disable-next-line no-any
-  const ComponentWithPlatform = (props: any) => (
+export function withPlatform<
+  TPlatforms extends string = string,
+  TProps extends IWithPlatformProps<TPlatforms> = IWithPlatformProps<TPlatforms>
+>(Component: React.ComponentType<TProps>): React.StatelessComponent<IWithPlatformProps<TPlatforms>> {
```

Comment:
How will this affect consuming this function?

Reply:
Since the new type parameter has a default value, you should be able to consume the function the same way as before and only specify a different props type if you want to.